php/class-post-type.php

```diff
@@ -25,6 +25,20 @@ class Post_Type {
 	 */
 	public $menu_position_key = 'menu_position';
 
+	/**
+	 * Sub menu meta key.
+	 *
+	 * @var string
+	 */
+	public $sub_menu_key = 'sub_menu';
+
+	/**
+	 * Menu parent meta key.
+	 *
+	 * @var string
+	 */
+	public $menu_parent_key = 'menu_parent';
```

Comment:
Why are we doing `menu_parent` instead of `parent_menu`? To me, this reads nicer, and matches better with `sub_menu`.